charts/ratify/crds/store-customresourcedefinition.yaml

```diff
@@ -13,16 +14,23 @@ spec:
     singular: store
   scope: Cluster
   versions:
-  - name: v1alpha1
+  - deprecated: true
+    deprecationWarning: v1alpha1 of the eraser API has been deprecated. Please migrate
+      to v1beta1.
+    name: v1alpha1
     schema:
       openAPIV3Schema:
         description: Store is the Schema for the stores API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
           kind:
-            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
```

Comment:
it will be good to test some migration scearnios. E.g Customer had old version of ratify install, by installing new ratify, we will deploy beta resources, is it possible for the to end up with two stores ( 1 beta , 1 alpha)

Reply:
Good question! The new CRDs would be applied in a new install or upgrade. And all resources of old versions would be upgraded to the latest version, so there wouldn't exist 2 stores/verifiers in 2 versions.